pimd/pim_cmd.c

```diff
@@ -1479,8 +1459,6 @@ static void pim_show_interfaces(struct pim_instance *pim, struct vty *vty,
 			vty_out(vty, "%9d\n", json_object_get_int(json_tmp));
 		}
 	}
-
-	json_object_free(json);
```

Comment:
Same comment as Donald's above. This is a really weird pattern to use JSON object for regular printing here.

Reply:
Yeah, didn't notice this loop inside non-JSON output... reverted.